See blog/see-blog.html

```diff
@@ -1,14 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" />
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    />
     <link rel="stylesheet" href="see-blog.css" />
     <title>Blog</title>
-</head>
-<body>
+  </head>
+  <body>
+      <!-- start the header section -->
+    <header>
+      <div class="left-side">
+        <i class="fas fa-chevron-left"></i>
+      </div>
+      <div class="right-side">
+        <img src="../images\love.png" alt="." />
+        <div class="more"><i class="fas fa-ellipsis-v"></i></div>
+      </div>
+    </header>
+    <!-- end the header section -->
+    <br />
     
-</body>
-</html>
\ No newline at end of file
+    <section class="top-par">
+      <p>The Inspiring People Helping With Coronavirus Efforts</p>
+    </section>
+    
+    <section class="author">
+      <div class="person"><img src="../images\Ellipse 36 (1).png" /></div>
```

Comment:
The `img` is too small, you need to make it bigger, especially in the desktop size